Snowdog_Components/components/Organisms/minicart-product/_minicart-product.scss

```diff
@@ -4,6 +4,7 @@
     display: flex;
     width: 100%;
     flex-flow: row nowrap;
+    flex-wrap: wrap;
```

Comment:
Could you update value of `flex-flow` instead of adding `flex-wrap`?

Reply:
Updated